src/teleop/gui/src/components/DriveMoteusStateTable.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="wrap">
+    <div>
+      <div>
+        <h3>Drive Moteus States</h3>
+      </div>
+      <table
+        class="tableFormat"
+        style="undefined;table-layout: fixed; width: 400px"
+      >
+        <colgroup>
+          <col style="width: 20px" />
+          <col style="width: 30px" />
+          <col style="width: 30px" />
+        </colgroup>
+        <thead>
+          <tr>
+            <th class="tableElement tableHeader">Motor</th>
+            <th class="tableElement tableHeader">
+              {{ moteusStateData.name[indices.FrontLeft] }}
+            </th>
+            <th class="tableElement tableHeader">
+              {{ moteusStateData.name[indices.FrontRight] }}
+            </th>
```

Comment:
Could we make this table element a component? There's a lot of code duplication here that could potentially be fixed using a v-for, but it may be easier if it's a really small component that gets passed in the array (either name, state, or error) and the two indices it needs. So the actual template of that component would only be like 9 lines long

Reply:
v-for would also be cool though if you think there's an easy way to do that